contrib/nginx-example.conf

```diff
@@ -6,6 +6,12 @@ server {
     server_name docket;
     server_name _;
 
+    location ~* ^/([a-f0-9]+/[a-z0-9_.-]*\.pcap)$ {
```

Comment:
*TODO* I'd like to discuss this more in chat.

Reply:
I didn't figure out the lighttpd configuration file, and if there's a better way to make the webserver host files in SPOOL_DIR, I'll be happy to make it so.